src/core/store/modules/lines/lines.selectors.ts

```diff
@@ -56,7 +57,8 @@ const selectLinesActionsStatusMap = (state: RootState) => state.lines.statusMap.
 const selectGetLinesStatus = (state: RootState) => state.lines.statusMap.getLines;
 const selectGetLinePageStatus = (state: RootState) => state.lines.statusMap.getLinePage;
 const selectGetUserLinesPositionsStatus = (state: RootState) => state.lines.statusMap.user.getUserLinePositions;
-const selectGetBorrowerPositions = (state: RootState) => state.lines.user.borrowerPositions;
+const selectBorrowerPositions = (state: RootState) => state.lines.user.borrowerPositions;
```

Comment:
@Seroxdesign I did not touch any of the previous code made by @Craigson to get borrower positions. 

However, this PR does build upon Craig's work getting the borrower positions and there are some redundant calls. Once you feel comfortable with the new query I have provided, I would like to deprecate the `getBorrowerPositions` query, or at the very least not call it with `dispatch` in the frontend to improve site performance.

Reply:
Did you test to see if this is working?